### PR TITLE
Pixi- Fix cls decimal values

### PR DESCRIPTION
**Pull request by robinvanopstal**

### CODE REVIEW

File: pixi/src/checks/PageExperienceCheck.js [hunk 1]
```diff
@@ -75,12 +75,12 @@ export default class PageExperienceCheck {
       proportion: METRICS_SCALES[id],
     };
 
-    if (metricOrigin.score < 0.5) {
-      data.category = Category.SLOW;
-    } else if (metricOrigin.score < 0.9) {
+    if (metricOrigin.numericValue <= METRICS_SCALES[id].fast) {
```
Comment (caroqliu) [inline pixi/src/checks/PageExperienceCheck.js:78]:
```suggestion
    if (metricOrigin.numericValue < METRICS_SCALES[id].fast) {
```

File: pixi/src/checks/PageExperienceCheck.js [hunk 2]
```diff
@@ -75,12 +75,12 @@ export default class PageExperienceCheck {
...
+      data.category = Category.FAST;
+    } else if (metricOrigin.numericValue <= METRICS_SCALES[id].average) {
```
Comment (caroqliu) [inline pixi/src/checks/PageExperienceCheck.js:80]:
```suggestion
    } else if (metricOrigin.numericValue < METRICS_SCALES[id].average) {
```

File: pixi/mocks/pageExperienceCheck/reportDataErrors.json [hunk 3]
```diff
@@ -40,7 +40,7 @@
 			}
 		},
 		"labData": {
-			"isAllFast": false,
+			"isAllFast": true,
```
Comment (caroqliu) [inline pixi/mocks/pageExperienceCheck/reportDataErrors.json:43]: Can you instead update the scores such that the `category` properties do not have to change?

Comment (robinvanopstal) [reply pixi/mocks/pageExperienceCheck/reportDataErrors.json:43]: Good point, thanks!

**Review by caroqliu — APPROVED** [review]

**Pull request merged**